esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -860,6 +860,29 @@ void EsdlServiceImpl::handleFinalRequest(IEspContext &context,
 
             soapmsg.append(req.str());
             soapmsg.append("</").append(tgtQueryName).append("></soap:Body>");
+
+            auto cfgGateways = tgtcfg->queryBranch("Gateways");
+            if (cfgGateways)
+            {
+                auto baseXpath = cfgGateways->queryProp("@transformTarget");
```

Comment:
It might be a better reminder of what this attribute does if we call it the "legacyTransformTarget"?  Or if you can think of another good name that is more indicative of the exact format we are producing.  We are calling a very specific transform here, not a script-able one.

Reply:
Will change to "legacyTransformTarget" since I don't have a better idea.